src/Components/SmartComponents/SystemCves/SystemCveTableToolbar.js

```diff
@@ -41,24 +41,25 @@ const SystemCveToolbarWithContext = ({
         methods.openCves(isOpen, expandedRows, !isAllExpanded);
     };
 
-    const { cves, parameters, methods, selectedCves, selectedRowsRawData, isAllExpanded, canEditPairStatus } = context;
+    const { cves, systemCVEs, parameters, methods, selectedCves, isAllExpanded, canEditPairStatus } = context;
     const { filter, advisory } = parameters;
     const selectedCvesCount = selectedCves?.length ?? 0;
-    const remediableCves = selectedRowsRawData.filter(cves => cves.attributes.remediation === ANSIBLE_REMEDIATION);
+    const remediableCves = selectedCves.filter(cves => cves.attributes.remediation === ANSIBLE_REMEDIATION);
 
     const selectOptions = useMemo(() => selectAllCheckbox({
         selectedItems: selectedCves,
         selectorHandler: methods.selectCves,
         items: cves,
+        rawItems: systemCVEs,
         fetchResource: ops => methods.fetchResource({ ...parameters, ...ops }),
         multiRow: true
-    }), [selectedCves, cves, parameters, methods]);
+    }), [selectedCves, cves, systemCVEs, parameters, methods]);
 
     const actions = ['',
         ...canEditPairStatus ?
             [{
                 label: intl.formatMessage(messages.editStatus),
-                onClick: () => methods.showStatusModal(selectedRowsRawData, true),
+                onClick: () => methods.showStatusModal(systemCVEs.payload?.data, true),
```

Comment:
This should only include selected CVEs, not all CVEs

Reply:
Oh god yes, you are right, Friday brain...

Reply:
Fixing this also fixed the edit status bug :)